src/main/java/dev/openfeature/sdk/MutableStructure.java

```diff
@@ -108,4 +110,17 @@ public Map<String, Object> asObjectMap() {
                 e -> convertValue(getValue(e.getKey()))
             ));
     }
+
+    /**
+     * Transform an object map to a {@link Structure} type.
+     *
+     * @param map map of objects
+     * @return a Structure object in the SDK format
+     */
+    public static Structure mapToStructure(Map<String, Object> map) {
+        return new MutableStructure(
+            map.entrySet().stream()
+                .filter(e -> e.getValue() != null)
+                .collect(Collectors.toMap(Map.Entry::getKey, e -> objectToValue(e.getValue()))));
+    }
```

Comment:
Should this be made generic and be defined on the interface? We want to make sure we maintain as much functionality as possible between `MutableStructure` and `ImmutableStructure`